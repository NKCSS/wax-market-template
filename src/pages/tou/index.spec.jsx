import { render } from '@testing-library/react'
import TermsOfUsePage from './index.page'

describe('::TermsOfUsePage', () => {
    it('renders without crashing', async () => {
        const { container } = render(<TermsOfUsePage />)
        expect(container).toMatchSnapshot()
    })
})
