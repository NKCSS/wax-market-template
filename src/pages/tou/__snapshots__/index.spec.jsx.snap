// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`::TermsOfUsePage renders without crashing 1`] = `
<div>
  <div />
</div>
`;
